rakelib/check.rake

```diff
@@ -4,34 +4,56 @@
 # frozen_string_literal: true
 
 namespace :check do
-  desc 'Optimize images in modified files, or by path (rake image_optim path=path/to/dir/or/file).'
+
+  desc 'Optimize images in modified files, or by path (rake check:img path=path/to/dir/or/file).'
   task :image_optim do
-    puts 'Running image optimizer...'.magenta
+    puts
+    puts 'Checking images ...'.magenta
     path = ENV['path']
+
     unless path
+      puts 'Looking in uncommitted files ...'.blue
       modified_files = `git ls-files --modified --others --exclude-standard`.split("\n")
       deleted_files = `git ls-files --deleted`.split("\n")
       image_files_to_check = (modified_files - deleted_files).select { |file| File.extname(file) =~ /\.(png|jpg|jpeg|gif)/i }
-      abort 'Didn\'t find any modified files.'.blue if image_files_to_check.empty?
+      
+      next puts 'No images to check.'.magenta if image_files_to_check.empty?
+
       path = image_files_to_check.join(' ')
     end
+
     system "bin/image_optim --no-pngout --no-svgo --recursive #{path}"
   end
 
-  desc 'Check Markdown syntax in modified files or in a particular file or directory by path (e.g. path=src/mftf)'
+  desc 'Check Markdown syntax in modified files or in a particular file or directory by path (e.g. path=mftf)'
   task :mdl do
-    puts 'Running Markdown linter ...'.magenta
+    puts
+    puts 'Checking Markdown ...'.magenta
+    print 'List the rules: $ '.magenta
+    sh 'bin/mdl -l --style=_checks/styles/style-rules-dev'
```

Comment:
Shouldn't we be using `_checks/styles/style-rules-prod` instead?

Reply:
No, it should be `dev` in rake `check`. The dev style contains all the rules we want to enable. Since we cannot do this all-at-once, we are enabling them one-by-one in our CICD. But for local development, in the changed only files, we want to check for entire target style, which is defined in `dev`. This will allows us to decrease the amount of work that needs to be done for enabling the style on production.